presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -1732,6 +1734,36 @@ public void dropView(ConnectorSession session, SchemaTableName viewName)
         return tableNames.build();
     }
 
+    @Override
+    public Map<String, Object> getSchemaProperties(ConnectorSession session, CatalogSchemaName schemaName)
+    {
+        if (!filterSchema(schemaName.getSchemaName())) {
+            return Collections.emptyMap();
+        }
+        String warehouse = metastore.getConfigValue(WAREHOUSE.getVarname()).orElseGet(() -> WAREHOUSE.getDefaultVal().toString());
+        Optional<Database> db = metastore.getDatabase(schemaName.getSchemaName());
+        if (db.isPresent()) {
+            Optional<String> location = db.get().getLocation();
+            // this is currently the only property for Hive
+            // (only shown when it's not the default location)
+            if (location.isPresent() && !location.get().startsWith(warehouse)) {
+                return Collections.singletonMap(HiveSchemaProperties.LOCATION_PROPERTY, location.get());
+            }
+        }
+
+        return Collections.emptyMap();
+    }
+
+    @Override
+    public Optional<PrestoPrincipal> getSchemaOwner(ConnectorSession session, CatalogSchemaName schemaName)
+    {
+        if (!filterSchema(schemaName.getSchemaName())) {
```

Comment:
This is not responsibility of connector to filter schemas. It should be handled somewhere in `ShowQueriesRewrite`

Also if user is allowed to do show create schema then we don't need to filter schema here.

Reply:
This is for the information_schema and for Hive's sys schema.
(see `listSchemas`, `listTables` methods, etc.)

Reply:
That is to say: information_schema and sys have no owner as such, and they also do not have any properties.

Reply:
Please do not use `filterSchema` but just make sure that requested schema is one of these. And if it is the case I think we should throw an exception.

@findepi I think you are more familiar with sys schema. Are ok with my suggestion?

Reply:
See https://github.com/prestosql/presto/pull/3099#discussion_r406612194

`getSchemaOwner` should look similar to getSchemaProperties

(btw, i wasn't following -- did you consider combining them into single getSchemaMetadata operation?)